frame/evm/src/lib.rs

```diff
@@ -231,14 +229,26 @@ pub struct GenesisAccount {
 
 decl_storage! {
 	trait Store for Module<T: Trait> as EVM {
+		ChainId get(fn chain_id): u64;
+		GasPrice get(fn gas_price): U256;
```

Comment:
```suggestion
		ChainId get(fn chain_id) config(): u64 = 42;
		GasPrice get(fn gas_price) config(): U256;
```

Reply:
Thanks, you don't know macro magic until you do!